app/graphql/index.js

```diff
@@ -2,22 +2,27 @@ const { makeExecutableSchema } = require('graphql-tools'),
   types = require('./types'),
   inputs = require('./inputs'),
   users = require('./users'),
-  healthCheck = require('./healthCheck');
+  healthCheck = require('./healthCheck'),
+  albums = require('./albums');
 
-const typeDefs = [types, inputs, ...users.schemas, ...healthCheck.schemas];
+const typeDefs = [types, inputs, ...users.schemas, ...healthCheck.schemas, ...albums.schemas];
 
 module.exports = makeExecutableSchema({
   typeDefs,
   resolvers: {
     Query: {
       ...users.queries,
-      ...healthCheck.queries
+      ...healthCheck.queries,
+      ...albums.queries
     },
     Mutation: {
       ...users.mutations
     },
     Subscription: {
       ...users.subscriptions
+    },
+    Album: {
```

Comment:
whats the point of this? I think im not following your logic here

Reply:
This is for resolvers of the fields of Album, in this case is for the photos field of Album  type (see albums/queries.js)

Reply:
yes but I dont get your query structure, why an Album query outside of you mutations and queries?